citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -294,7 +294,7 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
     		String description = properties.getProperty( ResourceProperties.PROP_DESCRIPTION );
     		
      		String citationCollectionId = new String( resource.getContent() );
-    		CitationCollection collection = CitationService.getCollection(citationCollectionId);
+    		CitationCollection collection = CitationService.getInstance().getUnnestedCitationCollection(citationCollectionId);
```

Comment:
Good to get rid of covers but it's generally better avoid the `getInstance()` call by using `ComponentManager.get(Class)`